Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -88,3 +88,4 @@
 --rules trailingClosures
 --rules elseOnSameLine
 --rules sortTypealiases
+--rules forLoop
```

Comment:
Before merging I'll have to update this repo to point to a SwftFormat version that includes the new rule implementation